node/task.ts

```diff
@@ -17,7 +17,10 @@ export enum TaskResult {
     Failed = 1
 }
 
-export var _internal = {};
+let _internal = { } as any;
+if (process.env.TASKLIB_INPROC_UNITS) {
+    module.exports._internal = _internal;
```

Comment:
as long as TASKLIB_INPROC_UNITS is not defined during build, then _internal will not get exported and will not end up in the .d.ts

note, this piggybacks on the env var TASKLIB_INPROC_UNITS which is already used for other internal unit test behavior